modules/ROOT/pages/subqueries/subqueries-in-transactions.adoc

```diff
@@ -550,3 +550,83 @@ RETURN largeLists
 1+d|Rows: 2
 
 |===
+
+[[composite-databases]]
+== Composite databases
+`CALL { ... } IN TRANSACTIONS` can be used on link:{neo4j-docs-base-uri}/operations-manual/{page-version}/composite-databases[composite databases].
+Composite databases are virtual databases used for querying their constituents.
+The examples above are generally valid for composite databases, but they must be slightly
+modified to accommodate the nature of composite databases.
+
+The following two examples show how you can use `CALL { ... } IN TRANSACTIONS` on a composite database.
+The first query processes the file _friends.csv_ and creates `Person` nodes on all constituents.
+
+.friends.csv
+[source, csv, role="noheader" filename="friends.csv"]
+----
+1,Bill,26
+2,Max,27
+3,Anna,22
+4,Gladys,29
+5,Summer,24
+----
+
+.Query
+[source, cypher]
+----
+UNWIND graph.names() AS graphName
+LOAD CSV FROM 'file:///friends.csv' AS line
+CALL {
+  USE graph.byName( graphName )
+  WITH line
+  CREATE (:Person {name: line[1], age: toInteger(line[2])})
+} IN TRANSACTIONS
+----
+
+The second query removes all nodes from all constituents.
+
+.Query
+[source, cypher]
+----
+UNWIND graph.names() AS graphName
+CALL {
+  USE graph.byName( graphName )
+  MATCH (n)
+  RETURN id(n) AS id
+}
+CALL {
+  USE graph.byName( graphName )
+  WITH id
+  MATCH (n)
+  WHERE id(n) = id
+  DETACH DELETE n
+} IN TRANSACTIONS
+----
+
+The two examples demonstrate another useful feature of `CALL { ... } IN TRANSACTIONS`.
+Even though composite databases allow accessing multiple graphs in a single query, only
+one graph can be modified in a single transaction.
+`CALL { ... } IN TRANSACTIONS` offers a way of constructing queries which modify multiple graphs.
```

Comment:
I was very puzzled as to why the double invocation of `CALL` to delete, and then I saw your comment which clarifies. I don't think it's fair to assume the reader will understand without clarification, so I feel like we should explain _why_ you have to do this gymnastics and your first-thought-solution wouldn't work well.
Also, any special reason for using `id` rather than `elementId`?

Reply:
> Also, any special reason for using id rather than elementId?
good catch. It should use elementId

Reply:
>I was very puzzled as to why the double invocation of CALL to delete, and then I saw your comment which clarifies. I don't think it's fair to assume the reader will understand without clarification, so I feel like we should explain why you have to do this gymnastics and your first-thought-solution wouldn't work well.

I agree. But I think this might not be the best place. This is not behaviour specific to composite. 
The reason for the first CALL is the same why one of the  non-composite example looks like this:
```
MATCH (n)
CALL {
  WITH n
  DETACH DELETE n
} IN TRANSACTIONS
```
If you put the MATCH inside the CALL, it will be all one batch. It is something I personally did not understand. When I first used CALL IN TX. I thought the batching is driven by the rows produced in the CALL not rows coming in the CALL.

Reply:
OK. The doc actually says so if you read carefully "The number of input rows is set with the modifier `OF n ROWS` (or `OF n ROW`)."

Reply:
i agree, a short clarification (or a link?) may facilitate understanding the example

Reply:
I have added a note explaining why the nodes are matched outside the CALL IN TX. I have also added an equivalent note to the non-composite example. I feel that if this example needs an explanation, so does the non-composite one. Please have a look if you like the formulation.